src/field_theory/minimal_polynomial.lean

```diff
@@ -118,22 +243,6 @@ by { apply minimal_polynomial.dvd, rw [← is_scalar_tower.aeval_apply, minimal_
 
 variables [nontrivial β]
 
-/--The degree of a minimal polynomial is nonzero.-/
-lemma degree_ne_zero : degree (minimal_polynomial hx) ≠ 0 :=
-begin
-  assume deg_eq_zero,
-  have ndeg_eq_zero : nat_degree (minimal_polynomial hx) = 0,
-  { simpa using congr_arg nat_degree (eq_C_of_degree_eq_zero deg_eq_zero) },
-  have eq_one : minimal_polynomial hx = 1,
-  { rw eq_C_of_degree_eq_zero deg_eq_zero, convert C_1,
-    simpa [ndeg_eq_zero.symm] using (monic hx).leading_coeff },
-  simpa [eq_one, aeval_def] using aeval hx
-end
-
-/--A minimal polynomial is not a unit.-/
-lemma not_is_unit : ¬ is_unit (minimal_polynomial hx) :=
-assume H, degree_ne_zero hx $ degree_eq_zero_of_is_unit H
-
 /--The degree of a minimal polynomial is positive.-/
 lemma degree_pos : 0 < degree (minimal_polynomial hx) :=
 degree_pos_of_ne_zero_of_nonunit (ne_zero hx) (not_is_unit hx)
```

Comment:
This version of `degree_pos` can be removed if we generalize `degree_ne_zero` as above.